src/core/qgsvectorlayer.h

```diff
@@ -39,6 +40,7 @@
 #include "qgsaggregatecalculator.h"
 #include "qgsfeatureiterator.h"
 #include "qgsexpressioncontextgenerator.h"
+#include "qgsunittypes.h"
```

Comment:
If not strictly required to `#include ` headers in the header, do it in the source file. I.e. move these includes here to qgsvectorlayer.cpp